clai/README.md

```diff
@@ -51,10 +51,73 @@ Either way, running `clai` will start an interactive session where you can chat
 - `/multiline`: Toggle multiline input mode (use Ctrl+D to submit)
 - `/cp`: Copy the last response to clipboard
 
+## Web Chat UI
+
+Launch a web-based chat interface for your agent:
+
+```bash
+clai web --agent module:agent_variable
+```
+
+For example, if you have an agent defined in `my_agent.py`:
+
+```python
+from pydantic_ai import Agent
+
+my_agent = Agent('openai:gpt-5', system_prompt='You are a helpful assistant.')
+```
+
+Launch the web UI with:
+
+```bash
+clai web --agent my_agent:my_agent
+```
+
+This will start a web server (default: http://127.0.0.1:7932) with a chat interface for your agent.
+
+### Web Command Options
+
+- `--agent`, `-a`: Agent to serve in `module:variable` format
+- `--models`, `-m`: Comma-separated models to make available (e.g., `gpt-5,sonnet-4-5`)
+- `--tools`, `-t`: Comma-separated builtin tool IDs to enable (e.g., `web_search,code_execution`)
+- `--instructions`, `-i`: System instructions for generic agent (when `--agent` not specified)
+- `--host`: Host to bind the server to (default: 127.0.0.1)
+- `--port`: Port to bind the server to (default: 7932)
+
+### Using with Models and Tools
+
+You can specify which models and builtin tools are available in the UI via CLI flags:
+
+```bash
+# Generic agent with specific models and tools
+clai web -m gpt-5,sonnet-4-5 -t web_search,code_execution
+
+# Custom agent with additional models
+clai web --agent my_agent:my_agent -m gpt-5,gemini-2.5-pro
+
+# Generic agent with system instructions
+clai web -m gpt-5 -i 'You are a helpful coding assistant'
+```
+
+You can also launch the web UI directly from an `Agent` instance using `Agent.to_web()`:
+
+```python
+from pydantic_ai import Agent
+from pydantic_ai.builtin_tools import WebSearchTool
+
+agent = Agent('openai:gpt-5')
+
+# Use defaults
+app = agent.to_web()
+
+# Or customize models and tools
+app = agent.to_web(builtin_tools=[WebSearchTool()])
+```
+
 ## Help
 
 ```
-usage: clai [-h] [-m [MODEL]] [-a AGENT] [-l] [-t [CODE_THEME]] [--no-stream] [--version] [prompt]
+usage: clai [-h] [-m [MODEL]] [-a AGENT] [-l] [-t [CODE_THEME]] [--no-stream] [--version] [prompt] {web} ...
```

Comment:
This is the real help output right? We should be able to get things to render such that `web` is clearly a subcommand with its own options, right now it looks like it's a "web" word at the end of the regular command.